### ISSUE TITLE
Regression: Scripts don't work anymore (v3.16, Pro license)

### ISSUE THREAD

**Issue by steelsnake** [RealRaven2000/SmartTemplates#229]
extensions.smartTemplate4.allowScripts = true

Scripts like:
```
%{% 
    ( () => {
        var date = new Date();
        if (date.getHours() < 10)    {return "Good morning";}
        else                         {return "Hi";}
    } )() 
%}%
```
...get inserted into the email verbatim without getting parsed. Tested for new/reply/forwarded emails. SmartTemplates Pro v3.16, updated today.

**Comment by RealRaven2000** [comment]
ok, sorry about that. hopefully it should be easy to fix.

**Comment by RealRaven2000** [comment]
Can you try the version below?

[smartTemplate-fx-3.16.1pre1.zip](https://github.com/RealRaven2000/SmartTemplates/files/10982560/smartTemplate-fx-3.16.1pre1.zip)

you also have to make sure the config setting `extensions.smartTemplate4.sandbox=true` is set.

**Comment by RealRaven2000** [comment]
PS: to install the test version (3.16.1 prerelease 1) just download the zip file and drag into Thunderbird Add-ons Manager (no extracting necessary) - github doesn't allow attaching "xpi" files.

**Comment by steelsnake** [comment]
That fixed it! Thanks for the prompt reply and fix, very much appreciated.

**Comment by RealRaven2000** [comment]
Fixed in release 3.16.1 - published 21/03/2023
